rules/use-the-best-email-templates/rule.md

```diff
@@ -16,7 +16,9 @@ When sending repetitive emails, consistency across the organization is crucial.
 
 <!--endintro-->
 
-### Outlook Email Templates
+### Outlook Templates
+
+See here for more information on [Outlook Templates.](https://support.microsoft.com/en-au/office/create-an-email-message-template-43ec7142-4dd0-4351-8727-bd0977b6b2d1)

```

Comment:
Hi @Tanya-Leahy 

- [ ] 1. Please move this and all "More info..." to be after the examples

Reply:
Done - "More info...." moved under each example.